opds_import.py

```diff
@@ -966,6 +972,10 @@ def check_for_new_data(self, feed):
             # feed has changed.
             if record and record.status == CoverageRecord.TRANSIENT_FAILURE:
                 new_data = True
+                self.log.info(
```

Comment:
I added these log statements because it was difficult to understand why certain pages were being processed or not.